gstreamer-plugin/gstsvthevcenc.c

```diff
@@ -731,19 +784,18 @@ set_default_svt_configuration (EB_H265_ENC_CONFIGURATION * svt_config)
   svt_config->encoderBitDepth = 8;
   svt_config->encoderColorFormat = EB_YUV420;
   svt_config->compressedTenBitFormat = FALSE;
-  svt_config->profile = 1;
+  svt_config->profile = 2;
```

Comment:
If input profile is not 4 in Y444 case, bad parameter error returns from svt encoder.